### ISSUE TITLE
Please release your technical report. Need to read your ablation studies

### ISSUE THREAD

**Issue by stabgan** [tiiuae/Falcon-H1#10]
The Falcon H1 models are coming lower in ranking in the uncheatable eval, and in many discords discussions are going on regarding benchmaxxing.

https://huggingface.co/spaces/Jellyfish042/UncheatableEval

I believe this model is inspired by nvidia's Hymba and you are the first group who successfully trained it to compete with leading models.
But still I am curious why you chose mamba over RWKV-7 and what kind of ablation studies you did

**Comment by maksim-velikanov** [comment]
Thank you for your interest in the details behind Falcon-H1 models! We are working on the technical report and plan to release it in the coming weeks. Let us separately comment on your 3 different points:

1. **Relation with Hymba architecture**. We started working on Falcon-H1 shortly after the release of Falcon-Mamba-7B model in August 2024. We spend the first several months on extensive experimentation, including parallel vs. sequential hybrid design. With Hymba released just after launching Falcon H1's training, it was especially encouraging to see two teams independently converge to a similar parallel head design of the hybrid architecture.
2. **Relation with RWKV architecture**. There are several SSM architectures in the literature, including mamba1, mamba2, and different versions of RWKV. Following our experience with Mamba architecture during Falcon-Mamba-7B training, we decided to focus on careful exploration and optimization of various aspects of Mamba-based SSM design instead of trying other alternatives in the literature, for example RWKW. We believe this strategy helped Falcon-H1 models to reach or even surpass the performance of the top transformer models.
3. **Ranking in the uncheatable eval**. Upon a quick investigation of uncheatable eval leaderboard, we believe that it has certain biases towards the choice of pretraining data mixture that do not necessarily result in strong capabilities. Essentially, they use metrics such as bytes-per-character (BPC) that are very close to the cross-entropy loss of next token prediction minimized in the pretraining stage. Therefore, using a larger fraction of the data sources tested in the leaderboard (e.g. arxiv, wikipedia english, bbc news) in the pertaining data mixture will directly translate into strong performance on these metrics. However, the ability to very accurately predict the next token in a raw pretraining data source does not a priori give strong performance in downstream tasks or user experience. For example, SFT data, highly structured synthetic data, or simply raw pretraining data from other sources may be more efficient in driving the capabilities of the final model. We note, however, that this is our preliminary conclusion, and further investigations are required to fully understand the difference in performance of H1 models on classical benchmarks and the uncheatable eval leaderboard.

At the end of the day, interacting with the model and seeing how well it meets your actual needs is the ultimate evaluation. We welcome more hands-on evaluations—or vibe checks—of our Falcon-H1 models, and look forward to continued community discussion on how leaderboard metrics align with real-world performance and user experience.

**Comment by stabgan** [comment]
Also, why did you stop at 0.5B parameters ?
With the success of models like clinical mamba , smol LM2, and others can you also make a smaller model ? Maybe in the 150M parameter zone, for deployment to edge devices, or for training it in a very specific 1 task only ?

Curious if you tried going smaller and it just didn't perform well, or you never tried going smaller. Because if Falcon H1 0.5B is competing 7B models from 2024 according to your website, you can go smaller. So it competes Gemma3 1b and Qwen 3 0.6B models. This is a huge deal actually, if you have cracked that next gen architecture

**Comment by hypnopump** [comment]
I would also be interested in results for the UncheatableEval
